src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculate, DrawFigure {
```

Comment:
```suggestion
public class Circle extends Figure  {
```
в наслідниках не потрібно цього писати якщо батьківський клас вже реалізує інтерфейси